server/service/integration_logger_test.go

```diff
@@ -85,7 +85,7 @@ func (s *integrationLoggerTestSuite) TestLogger() {
 			assert.Equal(t, "/api/v1/fleet/config", kv["uri"])
 			assert.Equal(t, "admin1@example.com", kv["user"])
 		case 2:
-			assert.Equal(t, "info", kv["level"])
+			assert.Equal(t, "debug", kv["level"])
```

Comment:
What is the reasoning around logging `info` for every `/queries` request?

Reply:
Ah, I believe the reason `/queries` (and other) requests were logging `info` is that we were adding extras (`	logging.WithExtras(ctx, "name", query.Name, "sql", query.Query)`), and the `Log()` logic logged the request as info when that's the case.

Reply:
I think this was because of backwards compatibility only. Other than that, I don't remember.